lib/src/helpers/help_widget_view.dart

```diff
@@ -0,0 +1,17 @@
+/// It's defining a new type called [HelpWidgetView] with three possible values:
+/// `collapsed`, `main`, and `options`.
+///
+/// Helps to show different views
+enum HelpWidgetView {
+  /// It's a constant value of type [HelpWidgetView] that
+  /// has the value `collapsed`.
```

Comment:
This description is not really useful, is it?

How about we explain what this enum case is supposed to describe:
```suggestion
  /// A small, non-intrusive thumbnail view of the Widget.
```